homeassistant/components/onewire/sensor.py

```diff
@@ -195,7 +195,19 @@
 
 # 7E sensors are special sensors by Embedded Data Systems
 
-EDS_SENSORS: dict[str, list[DeviceComponentDescription]] = {
+EDS_SENSORS = {
```

Comment:
This line is probably what caused the need to rebase originally.
Can you please reset it with the type hint?

Reply:
Thanks

Reply:
Thanks